packages/e2e-tests/src/specs/dashboard/settings/adminUser/customFonts.js

```diff
@@ -103,10 +100,14 @@ describe('Custom Fonts', () => {
     await addCustomFont(OPEN_SANS_CONDENSED_LIGHT_ITALIC_URL);
     await addCustomFont(OPEN_SANS_CONDENSED_BOLD_URL);
     await addCustomFont(OPEN_SANS_CONDENSED_LIGHT_URL);
-    const fonts = await getFontList();
-    await page.click('div[role=listbox] [role=option]:last-child');
-    const selected = await getSelectedFont();
-    expect(selected.name).toStrictEqual(fonts[2].name);
+
+    const font = await page.$("div[role=listbox] [role=option]:last-child");
```

Comment:
Update to grab a reference to the font we want to work with and than look for  aria-selected = true